db/install.xml

```diff
@@ -1,19 +1,19 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<XMLDB PATH="mod/studentquiz/db" VERSION="20101203" COMMENT="XMLDB file for Moodle mod/studentquiz"
+<XMLDB PATH="mod/studentquiz/db" VERSION="20200116" COMMENT="XMLDB file for Moodle mod/studentquiz"
```

Comment:
@timhunt wow, I never ever touched that field, that must/should be updated constantly whenever you change the database scheme?